run-configurations/my-news-service-demo-01-test/application.properties

```
@@ -1,5 +1,5 @@
 ## Generic values useful for development
-settings.watt.server.ns.hideWmRoot=false
+settings.watt.server.ns.hideWmRoot=true
```

Please leave to false
```suggestion
settings.watt.server.ns.hideWmRoot=false
```